tests/integration-tests/configs/develop.yaml

```diff
@@ -4,13 +4,14 @@ test-suites:
 {{- common.OSS_COMMERCIAL_ARM.append("centos7") or "" -}}
 {% filter indent(2) %}
 {% include 'common/common.yaml' %}
+{% include 'common/rocky8.yaml' %} # FIXME: Overrides sections from common.yaml for Rocky8
 {% endfilter %}
   efa:
     test_efa.py::test_efa:
       dimensions:
         - regions: ["euw1-az1"]  # do not move, unless capacity reservation is moved as well
           instances: ["c5n.18xlarge"]
-          oss: ["alinux2"]
+          oss: ["rocky8"]
```

Comment:
we cannot change test_efa because we have OSU baseline results to check performances only for alinux2 at the moment

Reply:
The Baseline performance check is compared using the instance type like c5n.18xlarge